.circleci/config.yml

```diff
@@ -152,6 +152,65 @@ jobs:
           name: Build
           command: yarn run build
 
+  integration_tests:
```

Comment:
nit: I'd prefer to see this named `test-integration` and have the existing `test` job renamed to `test-unit`.